### PR TITLE
Make some visual improvements to mcdu.html

### PR DESCRIPTION
**Pull request by NewoIsTaken**
### Description of Changes
- Instead of having buttons be "1", "2", "3", etc. changed to "—".
- Added "△" to "OVFY" button.
- Remove "&nbsp;", no need for them.
- Changed button from "-" to "+/-"

### Checklist:

* [x] My changes follow the Contributing Guidelines.
* [ ] Please have a main Developer test my changes before merging.

### CODE REVIEW

File: mcdu.html [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <!DOCTYPE html>
 <html>
+
 <head>
 	<title>MCDU</title>
-	<meta charset="utf-8"/>
```
Comment (merspieler) [inline mcdu.html:5]: This is actually the proper xml syntax. Else the tag is never closed.

Comment (NewoIsTaken) [reply mcdu.html:5]: This document is HTML5 (not XHTML/XML) so there is no need for that. You can find more about it in the HTML5 reference [here](https://dev.w3.org/html5/html-author/#void).

File: mcdu.html [hunk 2]
```diff
@@ -77,35 +78,111 @@
...
+		img {
+			display: block;
+		}
 	</style>
 </head>
+
 <body style="width: 100%">
 	<table class="enter" style="width: 92%">
 		<tr>
-			<td>&nbsp;</td>
```
Comment (merspieler) [inline mcdu.html:98]: This changed section breaks alignment with the canvas display

**Review by merspieler — CHANGES_REQUESTED** [review]
I can't find the faulty line but the LSKs don't work with this version.

File: mcdu.html (hunk 2 above)
Comment (NewoIsTaken) [reply mcdu.html:98]: Hmm... let me look into this.

Comment (NewoIsTaken) [reply mcdu.html:98]: Fix committed! ff7c4e93f5ec6e9e13a5d4eb2c2ead4a39be0443

Comment (merspieler) [reply mcdu.html:98]: Looks good, tho the buttons still don't work... not sure what's wrong with that.

**Comment by NewoIsTaken** [comment]
I only made formatting changes to the JS but I will look into why the buttons don't work anymore.

**Comment by NewoIsTaken** [comment]
I found what is breaking the buttons and will fix it.

**Comment by NewoIsTaken** [comment]
@merspieler I fixed the buttons, they should work now.

**Comment by NewoIsTaken** [comment]
Actually, found another bug.

**Comment by NewoIsTaken** [comment]
Fixed the aforementioned bug. Should be good to merge now...

**Comment by NewoIsTaken** [comment]
I have changed this to a draft, I plan on making much more improvements before merging.

**Comment by NewoIsTaken** [comment]
OK, I have changed my mind again. I think we can release this for now.

**Comment by NewoIsTaken** [comment]
I'm going to put my major remake in a different pull request.

**Review by merspieler — APPROVED** [review]
Works fine now....

If I'd be nit-picking, I'd complain about the inline CSS... (I'm doing it as well... so no real issue to me)

**Comment by NewoIsTaken** [comment]
@merspieler Don't worry, I'm going to redo the whole thing.

**Pull request merged**